renovate.json

```
@@ -0,0 +1,5 @@
+{
+  "extends": [
+    "config:base"
+  ]
```

If you want to handle github actions too
```suggestion
  ], 
  "github-actions": {
     "schedule": ["before 3am on the first day of the month"] 
  }
  
```

---

It works by default: the "what to expect" section already lists "Update actions/github-script action to v4".